lib/main.js

```diff
@@ -2,23 +2,41 @@ const FIRSTPAGE = require('./firstpage.js');
 const NONFIRSTPAGE = require('./nonfirstpage.js');
 const util = require('./util.js');
 
-const DEFAULT_USER_AGENT = 'Mozilla/5.0 (compatible; bingbot/2.0; +http://www.bing.com/bingbot.htm)'; // eslint-disable-line max-len
+const DEFAULT_USER_AGENT = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.117 Safari/537.36'; // eslint-disable-line max-len
+const DEFAULT_X_YOUTUBE_CLIENT_NAME = '1';
+const DEFAULT_X_YOUTUBE_CLIENT_VERSION = '1.20200613.00.00';
```

Comment:
kinda dislike hardcoding all this
still thinking about a way to get this information from server responses

Reply:
If that could be done, that would be ideal, but if not, I think this is the more likely solution